docs/2023/CARE/RedesignDoctorNotes/index.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 1
+title: Introduction
+slug: /2023/CARE/RedesignDoctorNotes
+---
+
+### Author
+Bhavik Agarwal
+
+### Contact Info
+<!-- - [Email](mailto:) -->
```

Comment:
add email

Reply:
I have made the requested changes. @photon0205